pkg/image/qemu.go

```diff
@@ -111,12 +113,18 @@ func (o *qemuOperations) ConvertQcow2ToRawStream(url *url.URL, dest string) erro
 	return nil
 }
 
-func (o *qemuOperations) Resize(image, size string) error {
+// QuantityToQemu translates a quantity string into a Qemu compatible string.
+func quantityToQemu(size resource.Quantity) string {
 	// size from k8s contains an upper case K, so we need to lower case it before we can pass it to qemu.
 	// Below is the message from qemu that explains what it expects.
 	// qemu-img: Parameter 'size' expects a non-negative number below 2^64 Optional suffix k, M, G, T, P or E means kilo-, mega-, giga-, tera-, peta-and exabytes, respectively.
-	size = strings.Replace(size, "K", "k", -1)
-	_, err := qemuExecFunction(qemuLimits, nil, "qemu-img", "resize", "-f", "raw", image, size)
+	stringSize := strings.Replace(size.String(), "K", "k", -1)
+	stringSize = strings.Replace(stringSize, "i", "", -1)
+	return stringSize
+}
+
+func (o *qemuOperations) Resize(image string, size resource.Quantity) error {
+	_, err := qemuExecFunction(qemuLimits, nil, "qemu-img", "resize", "-f", "raw", image, quantityToQemu(size))
```

Comment:
Oh, this is the only place we're using that conversion call?  Mabye we don't even need the wrapper, but anyway... it would be nice to give it a better name IMO.

Reply:
Its the only place right now, we need it for instance for the blank image as well, and probably in the import from registry.